crates/bevy_app/src/app.rs

```diff
@@ -750,6 +755,42 @@ impl App {
         self
     }
 
+    /// Adds a function that will be called when the runner wants to initialize rendering state
+    ///
+    /// A renderer init function `init_fn` is called only once by the app runner once it has
+    /// determined that the system is in a suitable state for rendering state to be initialized.
+    ///
+    /// For example on Android a runner may wait until the application has reached a 'resumed'
+    /// state with an associated surface view before trying to initialize rendering state.
+    ///
+    /// `add_render_init` is typically only used by Bevy integrated plugins (e.g. `RenderPlugin`)
```

Comment:
```suggestion
    /// `add_render_init` is typically only used by Bevy-internal plugins (e.g. `RenderPlugin`)
```

Reply:
Yeah this reads better to me. I think I borrowed the wording from another similar App API and maybe it'd be good to tweak that too at some point.